donrec.php

```diff
@@ -148,7 +148,7 @@ function donrec_civicrm_searchColumns($objectName, &$headers,  &$values, &$selec
     // ************************************
     // only offer rebook only if the user has the correct permissions
     if (CRM_Core_Permission::check('edit contributions')) {
-      $contribution_status_complete = (int) CRM_Core_OptionGroup::getValue('contribution_status', 'Completed', 'name');
+      $contribution_status_complete = (int) CRM_Core_PseudoConstant::getKey('CRM_Contribute_BAO_Contribution', 'contribution_status', 'Completed');
```

Comment:
or should this be `::getName`? (I couldn't find API refs for the old method about what the last param 'name' means).
From the code (this value being compared to the contribution's status_id and how these values look in my database) I feel getKey (i.e. getting the value/id of the option) should be right but I didn't fully understand the logic of the feature and how to test it.

Reply:
getKey is correct the previous method says to look up the Completed status by searching in the name column, getKey by default searches in the name column